docs/privacyidea.md

```diff
@@ -102,6 +102,26 @@ The following is a template configuration:
      * Optional.
      */
     'autoSubmitOtpLength' => '',
+    
+    /**
+     * If 'pollInBrowser' option should use a deviating URL, set it here. Otherwise, the general URL will be used.
+     * Required only by 'pollInBrowser'.
+     */
+     'pollInBrowserUrl' => 'https://your.privacyidea.server',
```

Comment:
I would probably comment the line like `//'pollInBrowserUrl' => 'https://your.privacyidea.server',`, because like this, people need to enter their url either twice or remove this line theirself.
Most of the time, the general URL will suffice and if not, you can "enable" this line in the config.

Reply:
o, thanks:) missed it:)

Reply:
[ea0b50e](https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/211/commits/ea0b50e8f0a4d9fcf376ee9f135625f4cedfbad7)